### ISSUE TITLE
Add example requests and responses to Bots, Integrations endpoints

### ISSUE THREAD

**Issue by czechboy0** [buildasaurs/XcodeServer-API-Docs#1]
- [x] Bots
- [ ] Bot Patch example
- [ ] Integrations

**Comment by cojoj** [comment]
If we write some tests this will be covered automatically... But we're missing all `POST` tests which is a big part of this 😕

**Comment by czechboy0** [comment]
Yeah definitely, just copying cassette contents from XcodeServerSDK tests will make this much easier. Hey, extra motivation :wink:
